dockstore-webservice/src/main/java/io/dockstore/webservice/resources/AbstractWorkflowResource.java

```diff
@@ -419,6 +430,19 @@ private LambdaEvent createBasicEvent(String repository, String gitReference, Str
         return updatedServices;
     }
 
+    /**
+     * Decide whether a gitReference is excluded, given a workflow/service's filters
+     * @param gitRefPath Path.of(gitReference) for glob matching with PathMatcher
+     * @param filters Filters specified for a workflow/service in .dockstore.yml
+     * @return
+     */
+    private boolean filterGitReference(final Path gitRefPath, final List<String> filters) {
```

Comment:
I always go back and forth in my head about the meaning of filter (filter in or filter out?), but in Java, streams (and JavaScript filter as well), a filter that returns true means it should be included, so I think you should either rename this function or keep the name and reverse what it returns.